robocop/checkers/misc.py

```diff
@@ -371,6 +371,30 @@
     ),
     "0920": Rule(
         rule_id="0920",
+        name="unused-variable",
+        msg="Variable '{{ name }}' is assigned but not used",
+        severity=RuleSeverity.INFO,
+        docs="""
+    Variable was assigned but not used::
+
+        *** Test Cases ***
+        Test
+            ${variable}    Keyword Call  # not used variable
```

Comment:
I am thinking about making the examples like a real-world use cases. Something like:

```
*** Keywords ***
Get Triangle Base Points
    [Arguments]       ${triangle}
    ${p1}    ${p2}    ${p3}    Get Triangle Points    ${triangle}
    Log      Triangle base points are: ${p1} and ${p2}.
    RETURN   ${p1}    ${p2}
```

In the above example, variable `${p3}` is never used.

Reply:
Definitely looks better - I have also started to use more real like examples somewhere else, it just reads better than ``Test, Keyword Call,,,`` dry stuff :)